src/lib/server/session.cpp

```diff
@@ -105,7 +106,8 @@ void Session::_handle_simple_query() {
   // A simple query command invalidates unnamed portals
   _portals.erase("");
 
-  const auto execution_information = QueryHandler::execute_pipeline(query, _send_execution_info);
+  const auto [execution_information, transaction_context] =
+      QueryHandler::execute_pipeline(query, _send_execution_info, _transaction_context);
```

Comment:
Use std::tie to avoid having both `transaction_context` and `_transaction_context`